Will Heikoff Documentation

```diff
@@ -57,4 +57,51 @@ Replaced “G:\MAX-Filer\Research\Labs-kbuzard-S18\CARR\California\Base Data\Lab
 	by Buzard et al. which sites the use of only 645 labs rather than the 687 labs given in Labs.shp
 Continuing troubleshooting issue with PVAL syntax issue on line 191
 
+6/25/18
+Continued troubleshooting issue with PVAL syntax on line 191 with Prof. Buzard
+Found issue with importing shapefile on line 20
+Resolved issue by using Windows’ Command Prompt to install pyshp, the python shapefile library. Accomplished this by 
+	changing the current directory to the C: drive, by using the function “c:”, then using the 
+	“\Apps-SU\Anaconda3>python –m pip install pyshp” function to install the pyshp library
+		Used “\Apps-SU\Anaconda3” as this is where the pip package is located
+Created new Python Path for shapefile, using the location of where pyshp was installed and stored by the command prompt
+Changed ‘sys.path.append’ location to "G:\MAX-Filer\Research\Labs-kbuzard-S18\kfunctions" in order to create path to import
+	further modules
+
+6/28/18
+Entered "G:\MAX-Filer\Collab\Labs-kbuzard-S18\Admin\CARR\California\Base Data\ZIPS.shp" to set areal features on line 44 to
+	adjust for movement of Labs-kbuzard-S18 folder
+Entered "G:\MAX-Filer\Collab\Labs-kbuzard-S18\Admin\CARR\California\Base Data\GoodLabs.shp" to set point features on line 45
+	to adjust for the movement of Labs-kbuzard-S18 folder
+Tried to install shapely module using Windows Command Prompt yet error arose finding shapely and installation could not be completed
+Tried to install geopandas module using Windows Command Prompt yet error arose finding shapely and installation could not be completed
+
+7/2/18 - Working on new K-function Local_3Stage file
+Received new Three Stage K-functions files from Jake Carr and Prof. Buzard and saved them to the 
+	“G:\MAX-Filer\Collab\Labs-kbuzard-S18\Admin\kfunctions\Three Stage K-functions\Three Stage K-functions” folder
+Changed sys.path.append on line 23 to “G:\MAX-Filer\Collab\Labs-kbuzard-S18\Admin\kfunctions\Three Stage K-functions\Three Stage
+	K-functions” to adjust for the directory where the new k-functions are located
+Changed zipcodes data on line 36 to “G:\MAX-Filer\Collab\Labs-kbuzard-S18\Admin\CARR\California\Base Data\ZIPS.shp"
+Changed points data on line 37 to “G:\MAX-Filer\Research\Labs-kbuzard-S18\CARR\California\Base Data\GoodLabs.shp"
+Changed the reference variable on line 43 to “Emp_Manufa"
+Changed ZIPFIELD variable on line 46 to “ZIP"
```

Comment:
For these 3-stage programs, we need to find an additional file with the block-level data. Let's get together to brainstorm about that, and go back to getting the 2-stage programs working for now.